apps/frontend/src/utils/math.ts

```diff
@@ -11,6 +11,8 @@ const DEFAULT_DECIMALS = 6;
 const DEFAULT_DECIMALS_SEPARATOR = '.';
 const DEFAULT_THOUSANDS_SEPARATOR = ',';
 
+export const RAY = 27; // 27 decimal places
```

Comment:
useless comment. 

If we need to clarify about it I'd put it in the name. i.e.

`export const RAY_DECIMALS = 27;
`